packages/base/src/client.ts

```diff
@@ -0,0 +1,377 @@
+import type {
```

Comment:
This file shouldn't miss any changes that `scaffold/client` has. GH doesn't recognize it's the refactored version of it probably because we didn't move that page, we recreated it from scratch.

So manually check all functions are handled here as well.